spec/factories/claim/base_claims.rb

```diff
@@ -187,7 +187,7 @@ def random_case_number
   [%w(A S T).sample, rand(1990..2016), rand(1000..9999)].join
 end
 
-def set_amount_assessed(claim)
+def assign_fees_and_expenses_to(claim)
```

Comment:
`assign_fees_and_expenses_to(claim)` is better described as `assess_fees_and_expenses_for(claim)` to convey the business logic